### PR TITLE
updated example histogram.py color pallette

### PR DESCRIPTION
**Pull request by robinokwanma**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by bryevdv** [comment]
Hi @robinokwanma thanks for the PR! Generally we have avoided pure primary colors in examples. We have a section in the docs dedicated to some palettes that were designed with accessibility in mind:

https://docs.bokeh.org/en/latest/docs/reference/palettes.html#accessible-palettes

Would you try to find a pair of colors from one of those palettes that also have good contrast (e.g. as reported by https://webaim.org/resources/contrastchecker/) ? 

Edit: Also, do you mind adding before / after screenshots to the OP? It will make it much easier for us to evaluate the changes more quickly

**Comment by robinokwanma** [comment]
I sure will. Thanks for your response.

File: examples/plotting/histogram.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 def make_plot(title, hist, edges, x, pdf, cdf):
     p = figure(title=title, tools='', background_fill_color="#fafafa")
     p.quad(top=hist, bottom=0, left=edges[:-1], right=edges[1:],
-           fill_color="navy", line_color="white", alpha=0.5)
-    p.line(x, pdf, line_color="#ff8888", line_width=4, alpha=0.7, legend_label="PDF")
-    p.line(x, cdf, line_color="orange", line_width=2, alpha=0.7, legend_label="CDF")
+           fill_color="blue", line_color="white", alpha=0.5)
+    p.line(x, pdf, line_color="red", line_width=3, alpha=0.7, legend_label="PDF")
+    p.line(x, cdf, line_color="orange", line_width=2, alpha=0.7, legend_label="CDF") #updated colors and line width
```
Comment (bryevdv) [inline examples/plotting/histogram.py]: This kind of comment should be omitted, a brand-new user will not have any knowledge of what was updated or when.

**Comment by bryevdv** [comment]
Hi @robinokwanma rather than bare primary colors, can you look for a pair of reasonably high contrast colors taken from one  of the existing built-in palettes:

https://docs.bokeh.org/en/latest/docs/reference/palettes.html#accessible-palettes

Also please update the OP to include before after screenshots. We will also need to update the gallery thumbnails for this example. You can look at this PR for an example of what is necessary: https://github.com/bokeh/bokeh/pull/12946

**Pull request closed without merging**